jbpm-human-task/jbpm-human-task-jpa/pom.xml

```diff
@@ -65,11 +65,6 @@
       <scope>provided</scope>
     </dependency>
 
-    <dependency>
-      <groupId>org.codehaus.jackson</groupId>
-      <artifactId>jackson-core-asl</artifactId>
-    </dependency>
```

Comment:
@MarianMacik I added one more comment to remove old and not used Jackson dependency as it then is packaged into kieserver/spring boot apps

Reply:
Ok, I think we are ready to merge here.